### PR TITLE
Submitting Final DAEA Project

### PR DESCRIPTION
**Pull request by dressl14**
Alaynna Dressler
Popovers still aren't working. I looked over the code, but nothing seems out of place.

### CODE REVIEW

**Comment by jabiggs13** [comment]
The reason the popovers aren't working is because part of the code is missing at the end. This happened with a few other students and it's an easy fix. You'll want to do all the editing directly on your github account, no need to do it via atom or the desktop client. 

First, go to the AA-template.html code in the sites folder. Copy all of the code after the big space - it will start with </div> and end with </body></html>. Copy all of that (including the </div>). Then go to your site.html and click on the edit button in the top right-hand corner - it's a pencil icon. Then go to the very end of you code and paste in that ending code right after your last Sources Cited <p>...</p> tags. Then confirm/save.

Basically what happened was that you were working off of the AA-template version where someone accidentally deleted the end of the html code so the website doesn't know how to build or finish.

I'll go ahead and close this pull request. Let me know once you do all this and if it works or not. I can go back in and check it.

Jack
